lib/fileset/default.nix

```diff
@@ -510,4 +516,107 @@ If a directory does not recursively contain any file, it is omitted from the sto
       # We could also return the original fileset argument here,
       # but that would then duplicate work for consumers of the fileset, because then they have to coerce it again
       actualFileset;
+
+  /*
+    Like [`gitTrackedWith`](#function-library-lib.fileset.gitTrackedWith) with `{}` as the first argument, therefore using the defaults.
+
+    Type:
+      gitTracked :: Path -> FileSet
```

Comment:
The parameter needs documentation, because it doesn't support just any path. Refer to the second example in the parameter doc.

Reply:
I guess I could read this in the referred "Like ..." reference, but it would be nice for this to just be pretty much complete.

Reply:
It does have a sentence of docs if you look above the function argument:

> The [path](https://nixos.org/manual/nix/stable/language/values#type-path) to a local working directory of the Git repository whose tracked files to include.

I guess I could add a mention that a git working directory must have a `.git`.

Reply:
How did I not see that comment. Suggestion incoming.